zer0-os/zOS
# feat: add member was added admin message when member joins room

### What does this do?
- adds add member admin message when member joins a room.

### Why are we making this change?
- to highlight to other users that a member has joined the room.

### How do I test this?
- open messenger > start group > add member using three dot menu > check admin message.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
  
  
<img width="968" alt="Screenshot 2023-12-12 at 10 13 46" src="https://github.com/zer0-os/zOS/assets/39112648/66fe4024-80f7-453f-a4df-add39bb57e4b">
